src/client/camera.h

```diff
@@ -221,6 +230,9 @@ class Camera
 	ItemStack m_wield_item_next;
 
 	CameraMode m_camera_mode = CAMERA_MODE_FIRST;
+	v3f m_camera_custom_position = v3f(0.0, 0.0, 0.0);
```

Comment:
```suggestion
	v3f m_camera_custom_position = v3f(0.0f, 0.0f, 0.0f);
```

Please also add the trailing `f` for other new floats that aren't specified with it already.

Reply:
or shorter:
```suggestion
	v3f m_camera_custom_position = v3f(0f, 0f, 0f);
```

Reply:
I don't think our code style allows that, @sfan5

Reply:
I followed ClobberXD's style suggestion in the latest commit (expanded to cover all the added default values)